SoftwareGuide/Latex/DevelopmentGuidelines/CreateAModule.tex

```diff
@@ -8,6 +8,9 @@ \chapter{How To Create A Module}
 dependencies on other modules, which it declares. A module is defined with
 CMake scripts that inform the build system of its contents and dependencies.
 
+The modularization effort significantly improves the extensibility of the
+toolkit and lowers the barrier of contribution.
```

Comment:
barrier of contribution -> barrier to contribution

Reply:
Done.